server/custom/reports/model.js

```diff
@@ -17,7 +18,7 @@ var ReportsSchema = new mongoose.Schema({
     isPublic: {type: Boolean},
     nd_trash_deleted: {type: Boolean},
     nd_trash_deleted_date: {type: Date},
-    selectedLayerID: {type: String}
+    selectedLayerID: {type: Object, ref: 'Layers'}
```

Comment:
It's not an `Object`, it's an `ObjectId`

https://mongoosejs.com/docs/schematypes.html#objectids

Reply:
and `ref` is only needed for population, which is not used.

Reply:
Probably related: when I save a report, selectedLayerId is saved as a string instead of an object id, and it does not appear on the reports page